cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
```

Comment:
Was it mentioned to make this field non-required?

Reply:
I decided by myself to do it, non-required because about fields mentioned almost nothing)